### ISSUE TITLE
"Show this thread" is hidden

### ISSUE THREAD

**Issue by cmdrfletcher** [dangeredwolf/ModernDeck#166]
I don't know if it is intended, but I like to see the info "Show this thread" beneath the tweets.

I help myself with the following customs CSS. Maybe you can implement a switch for the user to choose.

```css
.js-show-this-thread {
	display: inherit;
	color:var(--retweetColor)!important;
	text-align:center;
}
```
Keep up the good work!

Patrick

**Comment by dangeredwolf** [comment]
I'm looking into how to implement thread indicators in a more seamless way, and I want to know what you think of this?

![Screenshot_6226](https://user-images.githubusercontent.com/2636265/88344597-f06c7800-cd11-11ea-8622-909f2fafc195.png)

**Comment by dangeredwolf** [comment]
I've implemented this for ModernDeck 9.0. Thanks for the feedback!
